asciidoctorj-core/src/main/java/org/asciidoctor/extension/Reader.java

```diff
@@ -20,6 +20,20 @@
     int getLineNumber();
 
     /**
+	 * Get the name of the file of the current line.
+	 *
+	 * @return file name
+	 */
+	public String getFile();
```

Comment:
Also, could you please change these tabs to spaces?
(Sorry, but I am pretty opinionated on this 😁